docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,293 @@
+.. _LPM-51: https://ls.st/LPM-51
+.. _GitHub: https://github.com/lsst
+.. _LDM-294: https://ls.st/LDM-294
+
+.. _ccd-drafting:
+
+Drafting workflow
+=================
+
+Write change-controlled documents using :doc:`DM's standard workflow <../processes/workflow>`.
+That is:
+
+1. Create a ticket branch from ``master``.
+2. Work on the ticket branch.
+3. Have the ticket peer-reviewed.
+4. Rebase and merge the ticket branch to ``master``.
+
+Merging to ``master`` does not denote acceptance by a CCB.
+Instead, it adds to the changeset that will be included in the next CCB review (see next).
+
+.. _ccd-release:
+
+Releasing a new version from the master branch
+==============================================
+
+Follow these steps to submit a document to the CCB and release a new baselined version:
+
+1. Check out the head of the ``master`` branch and create a new DocuShare upload from the generated PDF.
+   See :ref:`ccd-docushare-upload`.
+
+2. Submit a request to the CCB.
+   The procedure depends on the CCB:
+
+   - For project documents (such as ``LPM``, ``LSE``), create an `LCR <https://project.lsst.org/groups/ccb/>`_ with a pointer to the new document version in DocuShare.
+   - For DM documents (``LDM`` and ``DMTR``), create an :ref:`RFC <decision-making-rfc-creating>` with a pointer to the new document in DocuShare.
+     Set the JIRA state to "flagged" to notify the DM CCB.
+
+3. Create a release branch based off the same commit as the DocuShare tag:
+
+   - For a project document:
+
+     .. code-block:: bash
+
+        git checkout -b tickets/LCR-<N>
+        git push -u
+
+   - For a DM document:
+
+     .. code-block:: bash
+
+        git checkout -b tickets/RFC-<N>
+        git push -u
+
+   Replace ``<N>`` with the LCR or RFC number.
+
+4. When the CCB responds, they may ask for changes.
+   For minor and straightforward changes, you may commit changes to the release branch.
+   For more complex changes, or when multiple people  are working in parallel to address requests, use a ticket branch-based workflow instead.
+   Create the ticket branch from the head of the release branch and merge back to the release branch.
+
+   When issues are addressed, notify the CCB:
+
+   - For a project document, :ref:`create a new DocuShare upload <ccd-docushare-upload>` and notify the CCB.
+
+   - For a DM document, create a comment on the RFC confirming the changes and link to the ``https://<handle>.lsst.io/v/RFC-<n>`` landing page for the release branch.
+     You don't need to create intermediate DocuShare versions for the DM CCB.
+
+   Repeat this step for each round of CCB feedback.
+
+5. When the CCB approves document you create a release:
+
+   1. Commit to the head of the release branch with the following changes:
+
+      - Update `document's change record <https://lsst-texmf.lsst.io/lsstdoc.html#document-preamble>`_.
+        The Project librarian or DM release manager, through the CCB, determines the document's semantic version.
```

Comment:
Rob will also check the content of the change record before accepting the LSE/LPM document revision so you may need to iterate this step.